notmuch/nm_db.c

```diff
@@ -302,8 +302,13 @@ void nm_db_longrun_done(struct Mailbox *m)
 {
   struct NmAccountData *adata = nm_adata_get(m);
 
-  if (adata && (nm_db_release(m) == 0))
-    mutt_debug(2, "nm: long run deinitialized\n");
+  if (adata) {
```

Comment:
Please move the brace to a new line to match the coding style :)

Reply:
I've just done that.